src/ai/backend/manager/models/vfolder.py

```diff
@@ -1123,15 +1123,21 @@ def from_row(cls, ctx: GraphQueryContext, row: Row | VFolderRow) -> Optional[Vir
         if row is None:
             return None
 
+        def _get_field(name: str) -> Any:
+            try:
+                return row[name]
+            except sa.exc.NoSuchColumnError:
+                return None
+
         return cls(
             id=row["id"],
             host=row["host"],
             quota_scope_id=row["quota_scope_id"],
             name=row["name"],
             user=row["user"],
-            user_email=row["users_email"] if "users_email" in row else None,
+            user_email=_get_field("users_email"),
             group=row["group"],
-            group_name=row["groups_name"] if "groups_name" in row else None,
+            group_name=_get_field("groups_name"),
```

Comment:
Let's merge this for now, but please consider revamping whole column lookup logic as guarding `NoSuchColumnError` like here may confuse future maintainers when they accidentally failed to include column to their SELECT query.